### ISSUE TITLE
onclick firing on wrong elements

### ISSUE THREAD

**Issue by cdeutmeyer** [hyperapp/hyperapp#161]
I'm working on a [personal project](http://hyperapp-deutmeyer.rhcloud.com) built with Hyperapp and running into issues where click events are firing on elements other than the one I click on. I'm not sure I could re-create the problem in a codepen but it's pretty easy reproduce in the project. Basically if you click around and select rando stuff in the tree then type some junk in the search a couple times. Then click around rando (not just at the top) in the tree it happens. I can see the action is being fired multiple times from different tree node elements, and the tree selected item starts getting confused. 

Codemapping is turned on and the important files are codeNavigatorTree.js and model.js. 
If it's super lame to ask to peek at my code, I apologize and can maybe put together a codepen to try and illustrate the problem.

I'm a big fan of HyperApp and love what you guys are doing. Thanks!

**Comment by dodekeract** [comment]
Possible duplicate of #160.

**Comment by jbucaran** [comment]
@cdeutmeyer Yep, sorry! I've fixed this issue, but I was still reviewing #141, so I was hoping I could release everything in 0.8.0.
